plugins/kubernetes/app/models/kubernetes/role.rb

```diff
@@ -19,5 +19,13 @@ def label_name
     def ram_with_units
       "#{ram}Mi" if ram.present?
     end
+
+    def has_service?
+      service_name.present?
```

Comment:
Yeah, I'm not entirely happy with this either.  It's important that the name of the service stay consistent, because the service name is the value that defines the DNS lookup and the ENV vars that other applications will use. Also, it's important that service names be unique, so we should add a check to make sure no two `Kubernetes::Roles` in Samson try using the same service name. That's why I want it to be a column in the database.

But if someone makes a change to to the template file and gives the service a different name, what should we do?

I' still puzzling over this.